### PR TITLE
Enochtsang/sys 7/readme revamp

### PR DESCRIPTION
**Pull request by enochtsang**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+. link:https://yarnpkg.com/lang/en/docs/install/[Yarn] installs/manages all other dependencies
+
+To install all other dependencies, run `yarn`
+
+IMPORTANT: Please use yarn to manage your dependencies.
+
+=== Using Yarn
+
+The following commands should never be used in this repository.
```
Comment (Right2Drive) [inline README.adoc]: Maybe bold this so it's clear these are commands nobody should be running. I'm worried about skimmers that will just see the commands and instinctively run them :P

If you don't think this is necessary I'll adhere to your best judgement :+1:

Comment (enochtsang) [reply README.adoc]: I agree, I'll do that. Won't those commands fail without a `package-lock.json` file?

Comment (Right2Drive) [reply README.adoc]: No they shouldn't fail I don't think, and they will probably even generate a `package-lock.json` file (I don't know much about the file in general I didn't know until this project npm implemented a lock file themselves)

**Comment by Right2Drive** [comment]
@enochtsang good to go in 👍

**Pull request merged**